src/include/cell.h

```diff
@@ -153,21 +153,8 @@ struct __wt_cell_unpack {
 
     uint64_t v; /* RLE count or recno */
 
-    /* Value validity window */
-    wt_timestamp_t start_ts;         /* default value: WT_TS_NONE */
-    uint64_t start_txn;              /* default value: WT_TXN_NONE */
-    wt_timestamp_t durable_start_ts; /* default value: WT_TS_NONE */
-    wt_timestamp_t stop_ts;          /* default value: WT_TS_MAX */
-    uint64_t stop_txn;               /* default value: WT_TXN_MAX */
-    wt_timestamp_t durable_stop_ts;  /* default value: WT_TS_NONE */
-
-    /* Address validity window */
-    wt_timestamp_t oldest_start_ts;         /* default value: WT_TS_NONE */
-    uint64_t oldest_start_txn;              /* default value: WT_TXN_NONE */
-    wt_timestamp_t newest_start_durable_ts; /* default value: WT_TS_NONE */
-    wt_timestamp_t newest_stop_ts;          /* default value: WT_TS_MAX */
-    uint64_t newest_stop_txn;               /* default value: WT_TXN_MAX */
-    wt_timestamp_t newest_stop_durable_ts;  /* default value: WT_TS_NONE */
+    WT_TIME_WINDOW tw;    /* Value validity window */
+    WT_TIME_AGGREGATE ta; /* Address validity window */
```

Comment:
Are these stored on disk? If not, alphabetize.

Reply:
Fixed.